index.html

```diff
@@ -3,9 +3,74 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Document</title>
+    <title>Dev-collab</title>
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.7.2/css/all.min.css"
+      integrity="sha512-Evv84Mr4kqVGRNSgIGL/F/aIDqQb7xQ2vcrdIwxfjThSH8CSR7PBEakCr51Ck+w+/U6swU2Im1vVX0SVk9ABhg=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Big+Shoulders+Stencil:opsz,wght@10..72,100..900&family=Electrolize&family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&family=Roboto:ital,wght@0,100..900;1,100..900&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
   </head>
   <body>
-    <h1></h1>
+    <!-- header section -->
+    <header>
+      <div class="Header-top">
+        <div class="Icon1">
+          <ul>
+            <li>
+              <i class="fa-solid fa-calendar"></i>
+              <a href="#">12 Chaitra, 2081</a>
+            </li>
+          </ul>
+        </div>
+        <div class="Social-links">
+          <ul>
+            <li>
+              <i class="fa-brands fa-facebook"></i>
+              <i class="fa-brands fa-instagram"></i>
+              <i class="fa-brands fa-twitter"></i>
+              <i class="fa-brands fa-linkedin"></i>
+              <i class="fa-brands fa-youtube"></i>
+            </li>
+          </ul>
+        </div>
+      </div>
+      <div class="logo">
+        <img
+          src="Assests/Muted Rainbow Hexagon Thinking Graphic Organizer (1) 1.png"
+          alt="logo"
+        />
+      </div>
+
+      <nav class="Navbar">
+        <div class="Nav_links">
+          <ul>
+            <li><a href="#">Homepage</a></li>
+            <li><a href="#">International</a></li>
+            <li><a href="#">National</a></li>
+            <li><a href="#">Sports</a></li>
+            <li><a href="#">Finance</a></li>
+            <li><a href="#">About Us</a></li>
+          </ul>
+        </div>
+        <div class="Button">
+          <div class="S_btn">
```

Comment:
use better classnames. classname should have meaning not just random words or letter